lua/autorun/client/cl_votestuff_init.lua

```diff
@@ -0,0 +1,73 @@
+if SERVER then return end;
+
+local filterPattern = "&#?[%w]+;"
+
+surface.CreateFont( "Question-font", {
+	font = "Century Gothic",
+	size = ScreenScale(7),
+	scanlines = 0,
+	antialias = true,
+	extended = true,
+})
+
+surface.CreateFont( "Anwser-font", {
+	font = "Century Gothic",
+	size = ScreenScale(6.5),
+	scanlines = 0,
+	antialias = true,
+	extended = true,
+})
+
+net.Receive("tv::SendTasks", function( len )
+		local time = net.ReadUInt(16);
```

Comment:
You don't need to change it as I have a thing that will fix this automatically, but for future reference - you're not using standard lua syntax, you're using retarded Garry syntax, which is considered amateur

- don't put ; at the end of lines
- use "not" instead of !
- use "and" instead of &&
- use "or" instead of ||